src/App.tsx

```diff
@@ -1,173 +1,76 @@
-                    >
-                      Lievijne Jans
-                    </a>
-                  </td>
-
-                  <td>f</td>
-                  <td>1542</td>
-                  <td>1582</td>
-                  <td>-</td>
-                  <td>-</td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+import { Table } from './components/table';
+import { getPeople } from './api';
+import { Person } from './types';
+
+export const App: React.FC = () => {
+  const [activePage, setActivePage] = useState('');
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    if (activePage === 'people') {
+      getPeople()
+        .then(data => {
+          return setPeople(data);
+        });
+    }
+  }, [activePage]);
+
+  return (
+    <HashRouter>
+      <div data-cy="app">
+        <nav
+          data-cy="nav"
+          className="navbar is-fixed-top has-shadow"
+          role="navigation"
+          aria-label="main navigation"
+        >
+          <div className="container">
+            <div className="navbar-brand">
+              <NavLink
+                className={`navbar-item ${activePage === 'home' ? 'has-background-grey-lighter' : ''}`}
+                to="/"
+                onClick={() => setActivePage('home')}
+              >
+                Home
+              </NavLink>
+
+              <NavLink
+                className={`navbar-item ${activePage === 'people' ? 'has-background-grey-lighter' : ''}`}
+                to="/people"
+                onClick={() => setActivePage('people')}
+              >
+                People
+              </NavLink>
+            </div>
+          </div>
+        </nav>
+
+        <main className="section">
+          <div className="container">
+            <Route exact path="/people">
+              <div className="block">
+                <div className="box table-container">
+                  <h1 className="title">People Page</h1>
+                  <Table people={people} />
+                </div>
+              </div>
+            </Route>
+
+            <Route exact path="/">
```

Comment:
react router v6 doesn't support exact anymore.

// old - v5 `<Route exact path="/" component={Home} />`
// new - v6 `<Route path="/" element={<Home />} />`

As stated in their documentation:

You don't need to use an exact prop on <Route path="/"> anymore. This is because all paths match exactly by default. If you want to match more of the URL because you have child routes use a trailing * as in <Route path="users/*">.